core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -56,43 +58,88 @@
 import java.util.concurrent.TimeUnit;
 
 import javax.annotation.Nullable;
+import javax.net.ssl.SSLSession;
 
 /**
  * Common utilities for GRPC.
  */
+// TODO(lukaszx0) replace all deprecated calls internally and remove deprecated fields in 1.0 (?)
 public final class GrpcUtil {
 
   /**
    * {@link io.grpc.Metadata.Key} for the timeout header.
    */
-  public static final Metadata.Key<Long> TIMEOUT_KEY =
+  public static final Metadata.Key<Long> TIMEOUT_METADATA_KEY =
```

Comment:
This class is in `internal`. We are free to change it on a whim. I'm fine with not changing the other call-sites in this PR, though. There are only a couple of users of it: `ClientCallImpl`, `ServerImpl`, and a (very hacky) test.